### ISSUE TITLE
CodeTour shows misleading "Insert Code" link

### ISSUE THREAD

**Issue by thpani** [informalsystems/quint#609]
<img width="317" alt="Screen Shot 2023-02-03 at 14 29 15" src="https://user-images.githubusercontent.com/82047/216615322-05eaefce-68a0-4edf-9171-6860b49c447e.png">

CodeTour shows misleading "Insert Code" links on Markdown code fences.
Clicking the link inserts the code into the spec.

It looks like this happens when code fences are augmented with a highlight language (as in ```scala).

**Comment by konnov** [comment]
Yeah, I also noticed that. Since CodeTour does not highlight the code anyways, maybe we should just remove the language name after backticks?

**Comment by thpani** [comment]
Yeah, that'd fix it:

https://github.com/microsoft/codetour/blob/1ef66d7f646452e7aa664b44e16c977711415c6f/src/player/index.ts#L56

Maybe we should open a feature request on CodeTour though, to make this link optional?
